Makefile

```diff
@@ -172,162 +172,162 @@ ako-gateway-api-docker: glob-vars
 # tests
 .PHONY: k8stest
 k8stest:
-	sudo docker run \
+	sudo docker run --privileged \
 	-w=/go/src/$(PACKAGE_PATH_AKO) \
 	-v $(PWD):/go/src/$(PACKAGE_PATH_AKO) $(BUILD_GO_IMG) \
 	$(GOTEST) -v -mod=vendor $(PACKAGE_PATH_AKO)/tests/k8stest -failfast -coverprofile cover-1.out -coverpkg=./...
 
 .PHONY: integrationtest
 integrationtest:
-	sudo docker run \
+	sudo docker run --privileged \
```

Comment:
@DixitAakash: As per my understanding with --privilage option, AKO container will run with same authority as that of host OS process having access to all underlying resources. Do we want to provide such access level to container? Do we have anyother alternative to it?

Reply:
@akshayhavile I had to add this flag due to `Operation not permitted` issue on VM where Sonarqube Jenkins job is run. I added it because I don't see any such concern here as these containers are created with **golang:latest** image just to run the go test command and then stopped.

Reply:
At what step we are getting issue? While creating files `cover.out`? or while running the container itself? If its permission issue for copying file, i think it will be good to give permission to those folders.